crypto/store/store_local.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Mm, it seems the team has different opinions...  I share @ekasper's opinion that the names of new files shouldn't have this kind of arbitrary abbreviation that are only there because we were restricted by 8.3 file name format in the past.